lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -192,7 +192,7 @@ export class SearchFilterComponent implements OnInit, OnDestroy {
     private parseFacetItems(context: ResultSetContext, configFacetFields, itemType): FacetField[] {
         return configFacetFields.map((field) => {
```

Comment:
field doesn't look the right name. after you do field.field. Maybe currentConfigFacetField could be a better name to avoid to have field.field